crt_portal/static/sass/custom/table.scss

```diff
@@ -20,11 +20,9 @@
   }
 }
 
-.tr-status-new {
```

Comment:
this is for the entire row of new forms, I think they are still supposed to be bolded